gatsby-theme-untied/src/components/Video/styles.module.styl

```diff
@@ -1,13 +1,21 @@
 @require '../../themes/standard/config/breakpoints'
 @require '../../themes/standard/config/colors'
 
-.video-section
-  margin 0
+.video-container
+  margin: 0
+  padding: 0
   width: 100%
-  height: 100%
   text-align: center;
 
-  .video-container
-    width: 50%
-    height: 100%
-    display: flex
+  .inner-container
+    padding: 0
+
+    .overlay
+      position: absolute
+      margin: 20% 45%
```

Comment:
I would prefer to set dimensions width and height for the play button. To center the icon it's possible to use `top: 50%` and `left: 50%` together with `margin-top: -HEIGHT / 2` and `margin-left: -WIDTH / 2`